keyboards/rart/rartpad/ratpad.c

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 Alabahuy
```

Comment:
This file needs to be renamed `rartpad.c`

Reply:
done